### PR TITLE
update extra-dev package of mathcomp-analysis

### PR DESCRIPTION
**Pull request by affeldt-aist**
@proux01

### CODE REVIEW

File: extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.dev/opam [hunk 1]
```diff
@@ -1,41 +1,45 @@
...
+synopsis: "An analysis library for mathematical components"
+description: """
+This repository contains an experimental library for real analysis for
+the Coq proof-assistant and using the Mathematical Components library."""
+
+build: [make "-j%{jobs}%"]
+install: [make "install"]
+depends: [
+  "coq" { (>= "8.13" & < "8.15~") | (= "dev") }
+  "coq-mathcomp-ssreflect" { (>= "1.12.0" & < "1.13~") | (= "dev") }
```
Comment (proux01) [inline extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.dev/opam:19]:
```suggestion
  "coq-mathcomp-ssreflect" { (>= "1.12.0" & < "1.14~") | (= "dev") }
```

File: extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.dev/opam [hunk 2]
```diff
@@ -1,41 +1,45 @@
...
+depends: [
+  "coq" { (>= "8.13" & < "8.15~") | (= "dev") }
+  "coq-mathcomp-ssreflect" { (>= "1.12.0" & < "1.14~") | (= "dev") }
+  "coq-mathcomp-fingroup" { (>= "1.12.0" & < "1.14~") | (= "dev") }
+  "coq-mathcomp-algebra" { (>= "1.12.0" & < "1.14~") | (= "dev") }
+  "coq-mathcomp-solvable" { (>= "1.12.0" & < "1.14~") | (= "dev") }
+  "coq-mathcomp-field" { (>= "1.12.0" & < "1.14~") | (= "dev") }
+  "coq-mathcomp-finmap" { (>= "1.5.1" & < "1.6~") | (= "dev") }
+  "coq-mathcomp-bigenough" { (>= "1.0.0") }
+  "coq-hierarchy-builder" { (>= "0.10.0") }
```
Comment (palmskog) [inline extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.dev/opam:26]:
```suggestion
  "coq" { (>= "8.13") }
  "coq-mathcomp-ssreflect" { (>= "1.12.0") }
  "coq-mathcomp-fingroup" { (>= "1.12.0") }
  "coq-mathcomp-algebra" { (>= "1.12.0") }
  "coq-mathcomp-solvable" { (>= "1.12.0") }
  "coq-mathcomp-field" { (>= "1.12.0") }
  "coq-mathcomp-finmap" { (>= "1.5.1") }
  "coq-mathcomp-bigenough" { (>= "1.0.0") }
  "coq-hierarchy-builder" { (>= "0.10.0") }
```
How about only going with lower bounds here in `extra-dev` @affeldt-aist @proux01? Then you shouldn't have to update the package as often.

File: extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.dev/opam [hunk 3]
```diff
@@ -1,41 +1,45 @@
 opam-version: "2.0"
-maintainer: "pierre-yves@strub.nu"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+version: "dev"
```
Comment (palmskog) [inline extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.dev/opam:3]:
```suggestion
```

File: extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.dev/opam (hunk 2 above)
Comment (proux01) [reply extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.dev/opam:25]: Very good point! (although for mathcomp, we can hope next release will be 2.0 so we'll have to update the lower bound anyway ;-) )

**Comment by palmskog** [comment]
@affeldt-aist it looks like the whole build fails on `coq.dev` due to `coq-mathcomp-bigenough.dev` being broken on Coq `master`. Do you still want me to merge? See for example: https://gitlab.com/coq/opam-coq-archive/-/jobs/1852652993

Do we want to ping bigenough maintainers?

**Comment by proux01** [comment]
@palmskog I'm opening a PR on bigenough, so let's wait a bit.

**Comment by proux01** [comment]
Here it is https://github.com/math-comp/bigenough/pull/5

**Comment by palmskog** [comment]
math-comp/bigenough#5 looks good to me. It would be nice to have it merged so we can test that this package works in CI, but again because of `extra-dev` I can just merge this PR if you want me to.

**Comment by proux01** [comment]
Thanks @palmskog , math-comp/bigenough#5 is now merged, feel free to relaunch the CI/merge at your discretion.

**Comment by palmskog** [comment]
@affeldt-aist @proux01 now the build process starts properly, but we are getting the following error:
```
# File "./theories/topology.v", line 1037, characters 0-67:
# Error: Anomaly "Uncaught exception Not_found."
# Please report at http://coq.inria.fr/bugs/.
```
Do you still want me to merge?

**Comment by proux01** [comment]
@palmskog the error doesn't seem OPAM related, so I would say let's merge this and I'll investigate upstream

**Pull request merged**